src/bin/vip.js

```diff
@@ -27,7 +27,7 @@ const runCmd = async function () {
 	cmd
 		.command( 'logout', 'Logout from your current session' )
 		.command( 'app', 'List and modify your VIP applications' )
-		.command( 'backup', 'Generate a backup for VIP applications' )
+		.command( 'backup', 'Generate a backup of an environment.' )
```

Comment:
Should this match the text above?

`Generate a new database backup of an environment.`

Reply:
Sooo... this touches on our grander strategic discussion around the patterns that our VIP-CLI command library should follow in order to be truly extensible. 
By adding the command `vip backup`, we should be doing so because we believe that we will be adding a group of subcommands related to `backup` (e.g., `db`, `media`). 
In good faith that this might be the direction that we are heading in for `vip backup`, we are choosing to describe that command in a more neutral manner.